documentation/snippets/dynamic_sync_table.ts

```diff
@@ -20,7 +20,7 @@ pack.addDynamicSyncTable({
   getSchema: async function (context) {
     let datasourceUrl = context.sync.dynamicUrl!;
     // TODO: Fetch metadata about the data source and get the list of fields.
-    let properties = {
+    let properties: any = {
```

Comment:
Could probably use `coda.ObjectSchemaProperties` for correctness. Are you intentionally trying to avoid that complexity?